src/HealthGPS/person.cpp

```diff
@@ -47,7 +47,9 @@ double Person::get_risk_factor_value(const core::Identifier &key) const {
     throw std::out_of_range("Risk factor not found: " + key.to_string());
 }
 
-float Person::gender_to_value() const {
+float Person::gender_to_value() const { return gender_to_value(gender); }
```

Comment:
I'm wondering whether these helpers should be member functions of the `core::Gender` `enum class` instead?

Reply:
Good shout.